components/measurement/PerformanceDetails.js

```diff
@@ -19,24 +20,29 @@ const PerformanceDetails = ({
     },
     {
       label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.MaxPing' }),
-      value: maxPing.toString() + ' ms'
+      value: isNdt7 ? `~${maxPing.toString()} ms` : `${maxPing.toString()}ms`
     },
     {
       label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.MSS' }),
       value: mss.toString()
     },
     {
-      label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.PktLoss' }),
+      label: isNdt7 ? intl.formatMessage({
+          id: 'Measurement.Details.Performance.Label.RetransmitRate',
+          defaultMessage: 'RetransmitRate'
+      }): intl.formatMessage({ id: 'Measurement.Details.Performance.Label.PktLoss' }),
       value: packetLoss.toString() + '%'
     },
-    {
-      label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.OutOfOrder' }),
-      value: outOfOrder.toString() + '%'
-    },
-    {
-      label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.Timeouts' }),
-      value: timeouts.toString()
-    }
+    ...(isNdt7 ? [] : //only adds outOfOrder and  timeouts if isNdt7 is false
+      [{
+          label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.OutOfOrder' }),
+        value: outOfOrder.toString() + '%'
+      },
+      {
+        label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.Timeouts' }),
+        value: timeouts.toString()
+      }]
+    )
```

Comment:
While this is a clever way to initialize `items`, to make it clear, lets separate them. Please add the last two entries to `items` inside an `if` condition using `items.concat()`.

Reply:
OK